src/providers/fault-count/__tests__/fault-count.spec.ts

```diff
@@ -139,4 +161,55 @@ describe('FaultCountProvider', () => {
     });
   });
 
+  describe('getSeriousFaultSumCountCatBE', () => {
+    it('should return the serious faults count', () => {
+      expect((faultCountProvider as any).getSeriousFaultSumCountCatBE(catBETestDataStateObject)).toBe(1);
+    });
+    it('should return the correct count of serious faults', () => {
+      const failedState: CatBEUniqueTypes.TestData = {
+        ...catBETestDataStateObject,
+        vehicleChecks: {
+          tellMeQuestions: [{
+            code: 'string',
+            description: 'string',
+            outcome: 'DF',
+          }],
+          showMeQuestions: [{
+            code: 'string',
+            description: 'string',
+            outcome: 'S',
+          }],
+        },
+        eyesightTest: {
+          complete: true,
+          seriousFault: true,
+        },
+      };
+      expect((faultCountProvider as any).getSeriousFaultSumCountCatBE(failedState)).toBe(2);
```

Comment:
showMeQuestions shouldn't have an S - they are all driving faults, and the logic for if a serious fault is calculated is determined elsewhere.

Reply:
So this test doesn't reflect a situation that would occur in the app.